source/chapters/appendix/settings_directory.rst

```diff
@@ -8,28 +8,26 @@ The Mixxx Settings Directory
 .. sectionauthor::
    ronso0 <ronso0@mixxx.org>
 
-The Mixxx settings directory contains all user data and settings of your Mixxx installation.
+The Mixxx settings directory contains all user data and settings of your Mixxx installation. If you want to make a backup of your library and all preferences copy the entire directory to a safe place.
 
 Location
 --------
 
-**Linux**
-
-    ``~/.mixxx/``
-
-    **Note:** make sure you have "Show hidden files" enabled in your file manager
-    in order to show all dot-files and directories.
+.. hint:: On Windows, the :file:`AppData` folder is hidden, as well as all files and
+          directories beginning with a dot '.' on Linux. So if you have not already,
+          you will need to set your file manager to show hidden files and folders.
 
 **Windows**
-
-    | Vista and up: :file:`%LOCALAPPDATA%\\Mixxx`
-    | XP and below: :file:`%USERPROFILE%\\Local Settings\\Application Data\\Mixxx\\`
-    | Type either of those into the location bar of a Computer or Folder window.
+ | Vista and up: ``%LOCALAPPDATA%\Mixxx``
+ | XP and below: ``%USERPROFILE%\Local Settings\Application Data\Mixxx``
+ | Type either of those into the location bar of a Computer or Folder window.
 
 **macOS**
+ | Mixxx 2.3: ``~/Library/Containers/org.mixxx.mixxx/Data/Library/Application Support/Mixxx``
```

Comment:
Should we also print the resolved path for all OS?
/Users/<username>/Library/Containers/org.mixxx.mixxx/Data/Library/Application Support/Mixxx

Reply:
No, this is copy-pasteable, the other one isn't.

Reply:
We can also remove it from the othewr OSes, but i don't really care.